mesonbuild/interpreterbase/decorators.py

```diff
@@ -786,7 +786,7 @@ def feature_check_class(self) -> T.Type[FeatureCheckBase]:
         pass
 
     def __init__(self, feature_name: str, feature_version: str,
-                 kwargs: T.List[str], extra_message: T.Optional[str] = None):
+                 kwargs: T.List[str], extra_message: str = ''):
```

Comment:
This isn't the only place that it is Optional[str], but I suspect the better solution is to make the one instance be initialized as None rather than changing the two instances to initialize as a blank string.

Reply:
I agree. Fixed.